pandas/tests/series/test_ufunc.py

```diff
@@ -456,8 +454,7 @@ def add3(x, y, z):
         ufunc(ser, ser, df)
 
 
-# TODO(CoW) see https://github.com/pandas-dev/pandas/pull/51082
-@td.skip_copy_on_write_not_yet_implemented
+@pytest.mark.xfail(reason="see https://github.com/pandas-dev/pandas/pull/51082")
```

Comment:
Is there a way to still test this with cow?

Reply:
no not at the moment

Reply:
Okay sounds good.